nexus/src/authz/omicron.polar

```diff
@@ -212,7 +233,9 @@ resource Project {
 	"modify" if "admin";
 
 	relations = { parent_organization: Organization };
-	"admin" if "admin" on "parent_organization";
+	"admin" if "collaborator" on "parent_organization";
+
+	"viewer" if "list_children" on "parent_organization";
```

Comment:
Similar to the other case above, without this, a Fleet or Silo Viewer couldn't see into the Project.